src/javascript/app/base/binary_pages.js

```diff
@@ -44,6 +44,7 @@ const FinancialAccOpening     = require('../pages/user/new_account/financial_acc
 const RealAccOpening          = require('../pages/user/new_account/real_acc_opening');
 const VirtualAccOpening       = require('../pages/user/new_account/virtual_acc_opening');
 const WelcomePage             = require('../pages/user/new_account/welcome_page');
+const WelcomePageCr           = require('../pages/user/new_account/welcome_page_cr');
```

Comment:
lets name it WelcomePageOnboarding?, because sooner or later this will also cover both cr and eu

Reply:
done